### ISSUE TITLE
程序退出

### ISSUE THREAD

**Issue by noname346** [z814081807/DeepNER#11]
我使用cpu进行训练，但是在trainer.py的第136行:
loss=model(**batch_data)[0]
程序异常退出了。

我用的参数：
gpu_ids=-1
bert_type='roberta_wwm'
mode='train'
task_type='crf'
loss_type='ls_ce'

是一定要用gpu吗，还是我的安装有问题？
tensorflow 2.4.1
torch  1.8.1+cpu
transformers 4.4.2

**Comment by z814081807** [comment]
> gpu_ids=-1

一般情况下gpu_ids=-1就可以用cpu了，你这可能版本的问题。另外cup训bert几乎不可能，太慢了，还是用gpu吧，可以去了解一下colab 免费使用GPU
